Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRootRenderer.cs

```diff
@@ -254,16 +253,18 @@ void OnShellSectionItemsChanged(object sender, NotifyCollectionChangedEventArgs
 
 		void LayoutHeader()
 		{
-			if (_header == null)
-				return;
-
-			CGRect frame;
-			if (Forms.IsiOS11OrNewer)
-				frame = new CGRect(View.Bounds.X, View.SafeAreaInsets.Top, View.Bounds.Width, HeaderHeight);
-			else
-				frame = new CGRect(View.Bounds.X, TopLayoutGuide.Length, View.Bounds.Width, HeaderHeight);
-			_blurView.Frame = frame;
-			_header.View.Frame = frame;
+			int tabThickness = 0;
+			if (_header != null)
+			{
+				tabThickness = HeaderHeight;
```

Comment:
`HeaderHeight` is constant. Why put its value in a local variable?

Reply:
because it's only changing tabThickness if the _header is not null otherwise the tabThickness just stays zero